scripts/us_fed/treasury_constant_maturity_rates/maturities.py

```diff
@@ -0,0 +1,8 @@
+'''
```

Comment:
Is there a reason this is a standalone file and not  just part of the main python file?

Reply:
Because both generate_mcf.py and extract_data.py use it

Reply:
Can we combine all 3 scripts? I think it's reasonable to fetch the data and generate the mcf and tmcf files in one command.